src/pip/_internal/cli/req_command.py

```diff
@@ -302,6 +302,9 @@ def make_requirement_preparer(
                 gone_in="22.1",
             )
 
+        if options.quiet:
+            options.progress_bar = "off"
```

Comment:
Not that the combination makes a lot of sense, but what would/should `pip install --quiet --progress-bar=on` do?

Reply:
So, the change is useless because you could just add --progress-bar=off to silence the output?

Reply:
Not useless since `--quiet` _should_ disable the progress bar. I’m only wondering what happens if the user actually wants the progress bar back for some reason.

Reply:
Allow it?

Reply:
Yeah but I think your patch here as-is would not allow it?

Reply:
Yea, im going to update the patch.

Reply:
@uranusjr but by default progress_bar = on

Reply:
Hmm. Perhaps we need to add and change the default to `auto` or something. Let’s do it in another PR though, in here we can make `--quiet` override `--progress-bar=on`.